README.rst

```diff
@@ -1,9 +1,14 @@
 GameServer - Host your own game server
 =======================================================
 
-GameServer is a `TurnKey GNU/Linux`_ appliance for hosting
-game servers on Linux. It provides a way of deploying game servers
-in minutes in cloud environments or local VMs.
+GameServer is a `TurnKey GNU/Linux`_ headless server appliance for hosting
+game servers on Linux. Choose from up to `100 different game servers`_ to
+deploy to the cloud or local VM in minutes.
+
+Host a game server in minutes. Supports installation of a huge range of game
+servers, including Counter-Strike (Source & other CS titles), Call of Duty
+(& other CoD titles), Minecraft, GTA: San Andreas, Quake III, Unreal Tournament
```

Comment:
I'd suggest changing the line about Call of Duty, since it's used both in the context of the first game and the franchise as a whole it reads weird.